mlflow/server/js/src/experiment-tracking/components/ExperimentView.css

```diff
@@ -373,7 +373,7 @@ span.error-message {
 }
 
 .ExperimentView .sort-select {
-    width: 200px;
+    width: 300px;
```

Comment:
Is it possible to dynamically size this according to the contents?

Reply:
I will take a look if it is possible to dynamically size this.

Reply:
@dbczumar Kindly take a look at the latest commits for dynamic width sizing and let me know if this looks good.